spec/util/bundler_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'spec_helper'
+
+RSpec.describe PDK::Util::Bundler do
+  before(:each) do
+    allow(PDK::CLI::Exec).to receive(:execute).and_return({exit_code: 0})
+  end
+
+  describe '.ensure_bundle!' do
+    context 'when there is no Gemfile' do
+      before(:each) do
+        allow(File).to receive(:file?).with(/Gemfile$/).and_return(false)
+      end
+
+      it 'does nothing' do
```

Comment:
Yes, technically no Gemfile is required. Practically, all well-maintained do.

The goal is to provide a template so that `pdk new` generated modules look very similar to what is currently "on the market", to the point where the pdk can handle unmodified supported and voxpupuli modules.

For less well maintained modules, having a `pdk upgrade` command applying a template on top of an existing module provides more value, as it'll reduce long-term friction of supporting/working with multiple models.